docs/fuel-101/transaction-model.mdx

```diff
@@ -3,6 +3,132 @@ title: Transaction Model
+  lang="rust"
+  lineStart="4"
+  lineEnd="8"
+/>
+
+1. **Script Transactions** include all other types of transactions that are not `Create` or `Mint` transactions.
+2. **Create Transactions** create new contracts on the Fuel network. This is the type of transaction that occurs when you run `forc deploy`.
+3. **Mint Transactions** are used when new coins are minted as part of a coinbase transaction. `Mint` transactions are created by the block producer and are not signed.
+
+## What data is included in a transaction?
+
+Transactions on the Fuel network utilize a UTXO model (add link to UTXO page).
+There are several types of data contained in a transaction:
+
+- [Transaction Policies](#transaction-policies)
+- [Transaction Witnesses](#transaction-witnesses)
+- [Transaction Inputs](#transaction-inputs)
+- [Transaction Outputs](#transaction-outputs)
+- [Transaction Receipts](#transaction-receipts)
+
+### Transaction Policies
+
+A transaction must set certain policies in order to be valid. The policies include the gas price, witness limit, maturity, and the maximum fee allowed.
+
+### Transaction Witnesses
+
+A transction must contain the signatures of all parties involved in the transaction. Signatures are a type of witness data. Witness data is malleable, which means that it is not included in the transaction hash calculation.
+
+### Transaction Inputs
+
+There are three parent types of transaction inputs:
+
+<CodeImport
+  file="../fuel-specs/src/tx-format/input.md"
+  lang="rust"
+  lineStart="4"
+  lineEnd="8"
+/>
+
+1. **Coin** inputs represent funds are are being spent.
+2. **Contract** inputs are used when a contract is being called.
+3. **Message** inputs are used for messages.
+
+### Transaction Outputs
+
+There are five types of transaction outputs:
+
+<CodeImport
+  file="../fuel-specs/src/tx-format/output.md"
+  lang="rust"
+  lineStart="4"
+  lineEnd="10"
+/>
+
+1. **Coin** outputs represent a transfer of an asset. This type of output is only used when a transaction does not have any malleable fields. This means that the recipient, asset, and amount is known before the transaction is fully executed.
+2. **Contract** outputs are created when a transaction changes the state of a contract.
+3. **Change** outputs are similar to coin outputs, but are used when the amount is not known.
+4. **Variable** outputs are similar to coin outputs, but the recipient, asset, and amount are not known before the transaction is fully executed.
+5. **Contract Created** outputs are used when a contract is deployed.
+
+### Transaction Receipts
+
+After a transaction is executed, a list of receipts that contain all of the output information is generated. For example, if a contract logs a value, or panics during execution, this information will be contained in the receipts list.
+
+### Transaction Pointer
+
+The transaction pointer is a unique identifier that is used to reference the location of itself within a block.
```

Comment:
This feels confusing. TxPointer refers to any transaction in any block. It only refers to the tx it contains in case of Mint, where it makes the specific Mint unique.